companies/kabeleinsdoku.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "kabeleinsdoku",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "Seven.One Entertainment Group GmbH",
```

Comment:
please add a `runs`  with "Kabel Eins Doku", to improve the value of this record